lnrpc/rpc.proto

```diff
@@ -740,7 +740,7 @@ message SendToRouteRequest {
     string payment_hash_string = 2;
 
     /// The set of routes that should be used to attempt to complete the payment.
-    repeated Route routes = 3;
+    Route route = 3;
```

Comment:
comment should reflect the change.

Reply:
Has been dealt with in https://github.com/lightningnetwork/lnd/pull/2521